docs/install/windows.md

```diff
@@ -1,37 +1,405 @@
+
+Note: Find path file to version 7 (`C:\Program Files\PowerShell\7` or `C:\Program Files\PowerShell\7-preview`) and create a shortcut for this PowerShell 7
+Windows will continue to use 5 version if we try open "PowerShell" 
+
+PS: A shortcut is one option. You could also add it to your path. 
+
+------------------------------------------------------------ 
 
 ## Build Environment installation
 
-You can follow the lines of the scripts from the `ci` folder. The scripts can also be launched as-is for local installation. Perform a git clone of the `nrn` repository.
+### Step 6: Git clone
+At PowerShell 7, go to 
+`C:\Users\User>` and create folder named “Neuron”: 
+
+```
+mkdir Neuron
+```
+
+Inside the "Neuron” folder, do: 
+
+```
+git clone https://github.com/neuronsimulator/nrn
+```  
+
+ 
+Note: `PS C:\Users\User\Neuron> git clone git@github.com:neuronsimulator/nrn nrn` may be an option
+ (or it may end up in a no permission issue).  
+  
+------------------------------------------------------------ 
+
+### Step 7: Downloading Dependencies
+
+As **administrator** from PowerShell 7 (right button and “Run as Administrator”) at `C:\Users\User\Neuron\nrn\ci` run: 
+```
+.\win_download_deps.cmd
+```
+ 
+Note: You will receive a number of messages in the form below; they do not indicate errors.
+
+```
+"C:\Users\User\Neuron\nrn>pwsh -command Invoke-WebRequest -MaximumRetryCount 4 -OutFile python-3.8.exe https://www.python.org/ftp/python/3.8.2/python-3.8.2-amd64.exe   || goto :error" 
+```
+ 
+------------------------------------------------------------ 
+
+### Step 8: Installing Dependencies
+
+As **administrator** from PowerShell 7, at `C:\Users\User\Neuron\nrn\ci` run: 
+```
+.\win_install_deps.cmd
+``` 
+
+This command will take a while to complete.
+  
+------------------------------------------------------------ 
+
+### Step 9: (If your Windows doesn't have a Windows Subsystem for Linux)
+
+Install Windows Subsystem for Linux:  wsl.exe --update  
+
+`wsl.exe --list --online` shows you your options and 
+`wsl.exe --install Ubuntu-24.04` (or any other) selects an option.
+  
+------------------------------------------------------------ 
+
+### Step 10: (If you are installing NEURON in a Virtual Machine) 
+
+Turn off your VM and at your computer as **administrator** put in PowerShell 7: 
+
+`
+Set-VMProcessor -VMName "NameOfYourVM" -ExposeVirtualizationExtensions $true
+`
+
+For example:
 
-From a powershell with **administrator** priviledges, run:
 ```powershell
-PS C:\workspace\nrn> .\ci\win_download_deps.cmd
+Set-VMProcessor -VMName "Windows 11 dev environment" -ExposeVirtualizationExtensions $true 
+```
+
+ 
+------------------------------------------------------------ 
+
+ 
+### Step 11: We need execute the “mingw64.exe” 
+So, look for "msys64", go to folder “msys64” and execute `mingw64.exe`.
+
+Via folder: open folder `C:\msys64` (probably) and double click in the executable `mingw64` 
```

Comment:
Via folder: open folder `C:\msys64` (probably) and double click the executable `mingw64`